github/util_v4_repository.go

```diff
@@ -3,22 +3,20 @@ package github
 import (
 	"context"
 	"encoding/base64"
+	"errors"
 
 	"github.com/shurcooL/githubv4"
 )
 
 func getRepositoryID(name string, meta interface{}) (githubv4.ID, error) {
 
-	// Interperet `name` as a node ID and return
-	exists, err := repositoryNodeIDExists(name, meta)
+	// Interpret `name` as a node ID
+	exists, nodeIDerr := repositoryNodeIDExists(name, meta)
 	if exists {
 		return githubv4.ID(name), nil
 	}
-	if err != nil {
```

Comment:
As described here: https://github.com/integrations/terraform-provider-github/issues/597#issuecomment-766705593 there are some repo names that are base64 decodable (`wiki`, `abcdefgh `) and as a result an error would be returned here. Instead of returning an error, go ahead and try finding the repo by repo name anyways as fallback logic.